src/main/scala/chiseltest/legacy/backends/verilator/VerilatorExecutive.scala

```diff
@@ -59,7 +59,7 @@ object VerilatorExecutive extends BackendExecutive {
     // - TestCommandOverride
     // - CombinationalPath
     val compiledAnnotations = (new ChiselStage).run(
-      elaboratedAnno :+ RunFirrtlTransformAnnotation(new VerilogEmitter)
+      elaboratedAnno ++: RunFirrtlTransformAnnotation(new SystemVerilogEmitter) +: VerilatorCoverage.CoveragePasses
```

Comment:
Are there no consequences to changing to SystemVerilog, or should it be a param

Reply:
I see Verilator as a firrtl simulator. Changing to SystemVerilog emission should not impact the behavior, i.e., given the same firrtl you should not be able to tell the difference.